### DISCUSSION TITLE
Get stock prices with date range?

### DISCUSSION THREAD

**Discussion by RandomCouch** [ahmedsakr/wstrade-api#85] [Q&A] [answered]
First of all thank you for creating this wonderful api wrapper, I was looking to display chart data of a stock but couldn't figure out a way to get stock value information within a specific date range, I can only query the current value, I'm wondering if this is currently a feature and I'm not figuring it out or if this is a planned feature that hasn't yet been implemented?

Thank you!

**Comment by ahmedsakr** [comment]
My pleasure - I am glad to read that `wstrade-api` can be of use to you :)

And nope, you're right that it is not currently supported, but I think it would be a great addition. I have opened an issue for it and I'll try to integrate it next week when I am on break. https://github.com/ahmedsakr/wstrade-api/issues/86

**Comment by ahmedsakr** [comment, marked as answer]
@RandomCouch This is now supported in `v1.4.0` with the quotes.`history` API. You can Read more about using it [here](https://github.com/ahmedsakr/wstrade-api/tree/main/docs/quotes#quotes-history).

**Reply by RandomCouch** [reply]
That is awesome, thank you!
